src/components/Profile/UserProfile.tsx

```diff
@@ -0,0 +1,8 @@
+import { useOurMedia } from '../../hooks';
+import { DesktopProfile } from './desktop';
+import { MobileProfile } from './mobile';
+
+export const UserProfile = () => {
```

Comment:
You should add TODO comment and create issue for conntecting that screen to API

Reply:
I don't see created issue, which should be linked in here

Also please answer in the conversation always after making changes